juntagrico/dao/memberdao.py

```diff
@@ -78,33 +79,33 @@ def members_for_create_subscription():
 
     @staticmethod
     def members_for_email():
-        return Member.objects.exclude(inactive=True)
+        return Member.objects.exclude(q_deactivated())
 
     @staticmethod
     def members_for_email_with_subscription():
-        return Member.objects.filter(subscriptionmembership__subscription__activation_date__isnull=False).exclude(
-            inactive=True)
+        return Member.objects.filter(MemberDao.has_subscription).exclude(
```

Comment:
`MemberDao.has_subscription` should check that member has not left the given subscription. It should also be aware of future dates. That's actually a bug from another feature.